src/Bridge/Symfony/Bundle/Command/SwaggerCommand.php

```diff
@@ -65,9 +70,15 @@ protected function configure()
      */
     protected function execute(InputInterface $input, OutputInterface $output)
     {
+        /** @var string $version */
+        $version = $input->getOption('spec-version');
+        if (!\in_array($version, ['2', '3'], true)) {
+            throw new InvalidOptionException(sprintf('This tool only support version 2 and 3 of the OpenAPI specification ("%s" given).', $version));
+        }
+        $documentationNormalizer = sprintf('v%ddocumentationNormalizer', $version);
         $documentation = new Documentation($this->resourceNameCollectionFactory->create(), $this->apiTitle, $this->apiDescription, $this->apiVersion, $this->apiFormats);
-        $data = $this->documentationNormalizer->normalize($documentation);
-        $content = $input->getOption('yaml') ? Yaml::dump($data, 6, 4, Yaml::DUMP_OBJECT_AS_MAP | Yaml::DUMP_EMPTY_ARRAY_AS_SEQUENCE) : (json_encode($data, JSON_PRETTY_PRINT) ?: '');
+        $data = $this->$documentationNormalizer->normalize($documentation);
+        $content = $input->getOption('yaml') ? Yaml::dump($data, 10, 2, Yaml::DUMP_OBJECT_AS_MAP | Yaml::DUMP_EMPTY_ARRAY_AS_SEQUENCE) : (json_encode($data, JSON_PRETTY_PRINT) ?: '');
```

Comment:
I've increased the depth before inlining and reduced the indentation to 2 spaces as the latest version is even more nested